hera_cal/io.py

```diff
@@ -321,6 +322,7 @@ def get_metadata_dict(self):
         '''
         antpos, ants = self.get_ENU_antpos()
```

Comment:
let's make the data_ants=False kwarg explicit here to safeguard us against pyuvdata api changes